vehicle/autoware_raw_vehicle_cmd_converter/src/node.cpp

```diff
@@ -86,11 +109,11 @@ RawVehicleCommandConverterNode::RawVehicleCommandConverterNode(
 
 void RawVehicleCommandConverterNode::publishActuationCmd()
 {
-  if (!current_twist_ptr_ || !control_cmd_ptr_ || !current_steer_ptr_) {
+  if (!current_twist_ptr_ || !control_cmd_ptr_ || !current_steer_ptr_ || !actuation_status_ptr_) {
```

Comment:
Some vehicles do not publish actiation_status. This is not necessary for vehicles that do not require the steering conversion. Please make a specific condition for that.

Reply:
thanks! fixed in https://github.com/autowarefoundation/autoware.universe/pull/8504/commits/0a660ef011c913acebdc84291856bdea4e366410